modules/Bio/EnsEMBL/Utils/Tree/Interval/Immutable.pm

```diff
@@ -0,0 +1,320 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+=head1 CONTACT
+
+  Please email comments or questions to the public Ensembl
+  developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+  Questions may also be sent to the Ensembl help desk at
+  <http://www.ensembl.org/Help/Contact>.
+
+=cut
+
+=head1 NAME
+
+Bio::EnsEMBL::Utils::Tree::Interval::Immutable
+
+=head1 SYNOPSIS
+
+
+=head1 DESCRIPTION
+
+Heavily inspired by https://github.com/tylerkahn/intervaltree-python
+
+=head1 METHODS
+
+=cut
+
+package Bio::EnsEMBL::Utils::Tree::Interval::Immutable;
+
+use strict;
+
+use Data::Dumper;
```

Comment:
This module isn't used anywhere in this file, is it?

Reply:
Fixed.